packages/ui/src/theme/css/component/switchField.scss

```diff
@@ -0,0 +1,70 @@
+.amplify-switchfield {
```

Comment:
It feels like more of these should be CSS variables, but I'm unsure given how specific they are to our Switch design... But perhaps they still should be in case we want to override them at a theme level. Any thoughts @hvergara @dbanksdesign?

Reply:
I've added a few more CSS variables, but some of the basic default styles are still being set directly.  Maybe a pass can be made on these styles as well as other defaults for other primitives based on the design decision that is made.